cocos/native-binding/impl.ts

```diff
@@ -157,3 +157,19 @@ export const native = {
     saveImageData: globalJsb.saveImageData,
     process: globalJsb.process,
 };
+
+if (NATIVE) {
+    ['blockingTimeout'].forEach((attr) => {
+        const desc = Object.getOwnPropertyDescriptor(globalJsb, attr);
+        if (desc) {
+            Object.defineProperty(native, attr, desc);
+        }
+    });
+
+    Object.defineProperty(native, 'onBlockingDetected', {
+        enumerable: true,
+        configurable: true,
+        get() { return globalJsb.onBlockingDetected; },
+        set(fn) { globalJsb.onBlockingDetected = fn; }
```

Comment:
Could we just use `value: globalJsb.onBlockingDetected` here?

Reply:
Developer may assign `native.onBlockingDetected` which should be synchorize to the `globalJsb` object